docs/source/release_notes.rst

```diff
@@ -10,6 +10,7 @@ Release Notes
         * Renamed ``ComponentGraph``'s ``get_parents`` to ``get_inputs`` :pr:`2540`
         * Removed ``ComponentGraph.linearized_component_graph`` and ``ComponentGraph.from_list`` :pr:`2556`
     * Documentation Changes
+        * Added documentation for `DaskEngine` and `CFEngine` parallel engines :pr:`2560`
```

Comment:
double tic marks on `DaskEngine` and `CFEngine` rather than single?

Reply:
ngl, I have no idea what the difference is :)